metals/src/main/scala/scala/meta/metals/Configuration.scala

```diff
@@ -28,8 +29,15 @@ object Configuration {
       enabled: Boolean = false,
       command: String = "test:compile"
   )
-  @JsonCodec case class Scalac(enabled: Boolean = false)
+  @JsonCodec case class Scalac(
```

Comment:
Might be nice to add a `def enabled: Boolean = completions.enabled || diagnostics.enabled`

Reply:
Added in 15398a1